src/main/webapp/app/components/tabs/ReviewHistoryTab.tsx

```diff
@@ -0,0 +1,89 @@
+import { GET_ALL_DRUGS_PAGE_SIZE } from 'app/config/constants/constants';
+import { FB_COLLECTION } from 'app/config/constants/firebase';
+import { HistoryCollection } from 'app/shared/model/firebase/firebase.model';
+import { downloadFile } from 'app/shared/util/file-utils';
+import { getAllGeneHistoryForDateRange, getHistoryEntryStrings } from 'app/shared/util/firebase/firebase-history-utils';
+import { componentInject } from 'app/shared/util/typed-inject';
+import { IRootStore } from 'app/stores';
+import { onValue, ref } from 'firebase/database';
+import React, { useEffect, useState } from 'react';
+import { Button, Input, Label, Row, Spinner } from 'reactstrap';
+
+function ReviewHistoryTab({ firebaseDb, drugList, getDrugs }: StoreProps) {
+  const [historyCollection, setHistoryCollection] = useState<HistoryCollection>(null);
+  const [startDate, setStartDate] = useState<string>('');
+  const [endDate, setEndDate] = useState<string>('');
+
+  useEffect(() => {
+    const unsubscribe = onValue(ref(firebaseDb, FB_COLLECTION.HISTORY), snapshot => {
+      setHistoryCollection(snapshot.val());
+    });
+
+    return () => unsubscribe?.();
```

Comment:
Keep it the way it is, but I think you can do it in a more high brow way.

```ts
return unsubscribe;
```
![image](https://github.com/oncokb/oncokb-transcript/assets/24961694/10aa903c-343b-4ab4-a553-cbc7c3e105a5)

Reply:
For sure, can start doing that in future